Sources/Fluent/Query/Filter/NodeUtilities/Filter+Node.swift

```diff
@@ -0,0 +1,136 @@
+import Foundation
+
+/// Conforms Filter to NodeRepresentable and adds a Node Initializer
+extension Filter: NodeConvertible {
+    public init(node: Node) throws {
+        let entityName: String = try node.get("entity")
+        let entityClass: AnyClass? = NSClassFromString(entityName)
```

Comment:
Yes, fortunately :) And it's the same behavior on all platforms.

```swift
import Foundation
class MyClass {
	
}

print(NSClassFromString("TempCode.MyClass"))
```

Tested on both Swift 3.1 & Swift 4.0 Dev @ https://swift.sandbox.bluemix.net/#/repl